lib/index.js

```diff
@@ -9,7 +9,13 @@ var contents = require('./contents')
 function toc(node, options) {
   var settings = options || {}
   var heading = settings.heading ? toExpression(settings.heading) : null
-  var result = search(node, heading, settings.maxDepth || 6)
+  var parentTypeWhiteList = settings.parents || []
+  var result = search(
+    node,
+    heading,
+    settings.maxDepth || 6,
+    parentTypeWhiteList
+  )
```

Comment:
This is getting a bit unwieldy, maybe pass `node, heading, settings` in, and do the defaults for `maxDepth` and `parentTypeWhiteList` in `search.js`?

Reply:
I have modified the function signature, commit [7487fe2](https://github.com/syntax-tree/mdast-util-toc/pull/53/commits/7487fe2d34850b386ddbde73735253b5dc55414e)